paywall/src/__tests__/utils/validators.test.js

```diff
@@ -56,6 +56,7 @@ describe('Form field validators', () => {
     expect(
       validators.isAccount('0X12345678901234567890abcdef0123ABCDEF0123')
     ).toBeFalsy()
+    expect(validators.isAccount(null)).toBeTruthy()
```

Comment:
first of all, what I'm about to say needs to be added as a comment

if the user logs out, then `null` is sent to the frontend. The account postMessage listener tries to validate it, sees `null`, and ignores it. This change ensures that even logged out user account is sent to the checkout UI.

As an alternative approach, what if we defined `0x0000...` as the default account for a logged out user instead of `null`? This would allow us to ensure we are using the same validator for all accounts, and the null account is still a can't-be-a-person account